src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Badging for app icons
+subhead: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen.
```

Comment:
See https://github.com/GoogleChrome/web.dev/pull/1622#discussion_r333151625 for note on the subhead. I think maybe the best examples of what we're aiming for with the subheads are like the ones on https://medium.com. Kind of like an extra hook or piece of context to pull the reader in.

Reply:
FYI @robdodson I added an item to the content checklist to check for this: https://github.com/GoogleChrome/web.dev/commit/be53611288bc181fecb7d9a8107c3ea58b66b48a

Reply:
great thank you!

Reply:
Shortened